### ISSUE TITLE
Specify platforms for which to generate icons

### ISSUE THREAD

**Issue by yfunk** [aeirola/react-native-svg-app-icon#53]
## Issue / Motivation
I have unfortunately ran into some issues related to `svg2vectordrawable` while using this tool, where it would only generate fallback PNG images instead of vector drawables, because certain SVG Attributes used by the Adobe XD and Illustrator export (`class`, `stroke-miterlimit`, `fill="none"`) are not supported by that library. 

I decided to move back to the Image Asset Studio on Android, but keep using this tool for generating iOS icons on non-Mac machines, since that works perfectly.

## Suggestion
It would be nice if there was an option to configure the platforms for which to generate launcher icons.

To achieve this the configuration available in `app.json` could be extended with a `platform` field, which allows users to specify an array of target platforms (`"android"` and `"ios"`), while defaulting to both.
```ts
{
  // [...]
  "svgAppIcon": {
    "foregroundPath": "./icon/icon-foreground.svg",
    "backgroundPath": "./icon/icon-background.svg",
    "platforms": [ "android", "ios" ]
  }
}
```

**Comment by aeirola** [comment]
This seems like a valid use case, and the proposed solution feels suitable for the job. I see you already managed to create a PR with an implementation. Awesome! Let's continue the discussion over there.

**Comment by aeirola** [comment]
This has now been released in v 0.4.0
